src/components/VDatePicker/VDatePicker.js

```diff
@@ -307,7 +332,7 @@ export default {
           min: this.minMonth,
           max: this.maxMonth,
           scrollable: this.scrollable,
-          value: (!this.value || this.type === 'month') ? this.value : this.value.substr(0, 7),
+          value: (!this.value || !this.value.length || this.type === 'month') ? this.value : (this.multiple ? this.value : [this.value]).map(val => val.substr(0, 7)),
```

Comment:
Split this line

Reply:
Actually maybe move it to a computed property, we should try to avoid too much computation in render functions.